proto/internal/temporal/server/api/historyservice/v1/request_response.proto

```diff
@@ -1147,3 +1147,17 @@ message UpdateActivityOptionsResponse {
     // Activity options after an update
     temporal.api.activity.v1.ActivityOptions activity_options = 1;
 }
+
+// (-- api-linter: core::0134::request-mask-required=disabled
+// (-- api-linter: core::0134::request-resource-required=disabled
+message PauseActivityRequest {
+    option (routing).workflow_id = "pause_request.workflow_id";
+
+    // Namespace ID of the workflow which scheduled this activity
+    string namespace_id = 1;
+
+    temporal.api.workflowservice.v1.PauseActivityByIdRequest pause_request = 2;
```

Comment:
nit: want to call this `frontend_request` since this is a more accurate description? Could have done this in other RPCs too but there's no need to change now IMHO.

Reply:
ok